startup/systemd-udevd_depmod.conf

```diff
@@ -1,2 +0,0 @@
-[Service]
-ExecStartPre=/sbin/depmod -a
```

Comment:
This file is not here by accident. It is part of customization feature where it is possible to concatenate a CPIO archive containing a new or updated driver onto the end of the install.img. Running depmod before udev allows the new driver module to be found and loaded.

While I don't particularly like the feature, it might be better to leave this file as is unless there is a good reason to remove it?

Reply:
My main reason was that I thought it was oly there for overriding the depmod config, but in my tests I recall it did not seem to work anyway: when changing the depmod config, this was not sufficient to get the config taken into account, whereas when I launched `depmod -a` by hand it did work.

When digging for any logs that would complain about some unrecognized systemd directive or similar I was also under the impression that any such logs would not reach `/var/log` - which annoyed me quite a bit but I did not take the time to dig this yet.

Reply:
I think `startup/depmod.conf ` is unnecessary as you said but `startup/systemd-udevd_depmod.conf` is unrelated and has a slightly arcane but valid use case.

Reply:
OK let's separate those issues then